longhaul/longhaul.md

```diff
@@ -0,0 +1,235 @@
+# IoT SDK Long Haul Technical Spec
+
```

Comment:
Can we expand upon the overall ##Goals of Longhaul testing here? i.e. what are the key results we're looking to see? off the top of my head:

1) % of successfully handled D2C messages
2) error types exposed?
3) tied to releases so that we can be immediately informed if a release has a regression?